pulp_container/app/tasks/sync_stages.py

```diff
@@ -129,10 +134,19 @@ async def run(self):
                 tag_dc = DeclarativeContent(Tag(name=tag_name))
 
                 content_data = json.loads(raw_data)
-                media_type = content_data.get("mediaType")
+                media_type = content_data.get("mediaType") or dl_res.headers["content-type"]
```

Comment:
https://testsuite.io/content-type-header
dl_res.headers.get("content-type")

Reply:
@lubosmj can you  extract this fix in a separate PR and mark it as bugfix so we can backport it later https://github.com/pulp/pulp_container/issues/883#issuecomment-1193869565